bin/concurrently.js

```diff
@@ -62,6 +62,11 @@ const args = yargs
             default: defaults.hide,
             type: 'string'
         },
+        'show-timings': {
```

Comment:
What do you think if this was only `--timings`?

Reply:
sounds good, I'll update